tags/key.go

```diff
@@ -26,8 +26,8 @@ type Key interface {
 	// Name returns the name of the key.
 	Name() string
 
-	// StringValue encodes the given value represented in binary to string.
-	StringValue(b []byte) string
+	// EncodeValue encodes the given value represented in binary to string.
+	EncodeValue(b []byte) string
```

Comment:
Doesn't this decode a value?

Reply:
This encodes the internal low-level representation into something else. For example, we may convert a float64 represented as a little endian byte slice into a hex encoded string.

Reply:
I am not sure I understand this API. BytesToString? Based on your example we will have a EncodeString(long). Also is better to read the name as a string EncodeByteToXXX or something similar.

Reply:
We will always use EncodeValue. The int64 key will implement a EncodeValue that converts the []byte into string. I cannot follow your comment here.

BytesToString is a Java conventions. Go users look for encoders/decoders terminology.

Reply:
ISTM this method is used to take a value stored as a []byte in a Tag, and pretty-print it.  It's not used for serialization/deserialization.  Just call it "StringFromValue" or "ValueToString" or similar, and document that it's for pretty-printing.

Similarly, the documentation for tags.Set.StringValue says the return value is "encoded as a string" -- that could probably be made clearer since that's not used for serialization either.